pusher-api/api-reference.md

```diff
@@ -0,0 +1,392 @@
+Pusher API reference
+=====================
+Pusher is a service that publishes information to the integrator's service. This information is data related to products, purchases, inventory etc. The purpose of this service is to ensure that integrators do not have to poll for data related to specific events. <br/>
```

Comment:
Why mention Pusher service when title is _Pusher API_? 
In my opinion it's confusing.

Reply:
We thought service and API are the same terms? If not, we can change that.